src/style/sheets/actor/character.scss

```diff
@@ -164,18 +274,95 @@
     app-character-paths-list {
         .path {
             position: relative;
-            border-radius: 0.3rem;
             margin-bottom: 1rem;
 
             display: flex;
             flex-direction: column;
 
-            border: 1px solid #463a47;
-            overflow: hidden;
-            background: #0b0a13;
+            background: var(--plotweaver-color-base-2);
             text-shadow: 0 0 0.5rem black;
             box-shadow: 0 0 0.5rem black;
 
+            .border {
```

Comment:
if this border block is exactly the same as the one above (or very similar to it and/or other declarations for borders) could we maybe make a SCSS mixin?